plugins/webp-uploads/settings.php

```diff
@@ -15,9 +15,9 @@
  * Registers setting for generating JPEG in addition to the selected modern format for image uploads.
  *
  * @since 1.0.0
- * @since n.e.x.t The setting was made more general to cover outputting JPEG as a secondary type. The "webp" option naming
+ * @since 2.0.0 The setting was made more general to cover outputting JPEG as a secondary type. The "webp" option naming
  *                was left unchanged for backward compatibility.
- * @since n.e.x.t The `perflab_modern_image_format` was added to enable selecting an output format.
+ * @since 2.0.0 The `perflab_modern_image_format` was added to enable selecting an output format.
```

Comment:
Not a blocker: Can we adjust it to use a single `@since`? I haven't seen two `@since` tags for a single release. Correct me if I'm wrong.